packages/button/src/Button.ts

```diff
@@ -140,4 +183,22 @@ export class Button extends SizedMixin(ButtonBase, { noDefaultSize: true }) {
             this.setAttribute('variant', this.variant);
         }
     }
+
+    protected override renderButton(): TemplateResult {
+        return html`
+            ${this.buttonContent}
+            ${when(this.isPending, () => {
+                import(
+                    '@spectrum-web-components/progress-circle/sp-progress-circle.js'
+                );
+                return html`
+                    <sp-progress-circle
+                        indeterminate
+                        static="white"
+                        label=${this.pendingLabel + 'progress circle'}
```

Comment:
Do we need the `progress circle` part? That forces un-translated content into the label.

Separately, is it even possible to get to the Progress Circle in the accessibility tree? Would it be better to use `aria-hidden="true"`, as this is namely a visible affordance for the change in label on the Button?

Reply:
you're right! this is just a visible affordance so it makes much more sense to have aria-hidden="true"